docs/user/rstudio/ide/guide/productivity/text-editor.qmd

```diff
@@ -2,43 +2,43 @@
 title: "Text Editor"
 ---
 
-The RStudio's source editor includes a variety of productivity enhancing features including syntax highlighting, code completion, multiple-file editing, and find/replace.
+RStudio's source editor includes a variety of productivity enhancing features including syntax highlighting, code completion, multiple-file editing, and find/replace.
 
-The RStudio also enables you to flexibly execute R code directly from the source editor. For many R developers this represents their preferred way of working with R. Working in the source editor makes it much easier to reproduce sequences of commands and to package commands for re-use as a function. These features are described in the [Executing Code](https://support.rstudio.com/hc/en-us/articles/200484448-Editing-and-Executing-Code-in-the-RStudio-IDE#executing) section below.
+RStudio also enables flexible execution of R code directly from the source editor. For many R developers this represents their preferred way of working with R. Working in the source editor makes it much easier to reproduce sequences of commands and to package commands for re-use as a function. These features are described in the [Executing Code](../code/execution.html) section of the user guide.
 
-Created from <https://support.rstudio.com/hc/en-us/articles/200484448-Editing-and-Executing-Code-in-the-RStudio-IDE>
+The core features listed below are accessible via the **Code Tools** "Magic Wand" dropdown menu in the **Source** pane.
 
-The core features listed below are accessible via the *Code Tools* "Magic Wand" in the **Source pane**.
-
-![](images/code-tools.png){fig-alt="Screenshot of the Code Tools dropdown" width="450"}
+![](images/code-tools.png){fig-alt="Screenshot of the Code Tools dropdown"}
 
 Most of these features have corresponding keyboard shortcuts, which also means they can be accessed via the Command Palette (<kbd>Ctrl/Cmd + Shift + P</kbd>).
 
+![](images/rename-in-scope.png){fig-alt="A screenshot of the Command Palette with the search limited to 'Rename' symbol in scope"}
+
 ## Code Completion
```

Comment:
Update section title casing in this file?

Reply:
changed to sentence case